app/main.py

```diff
@@ -2,4 +2,68 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords
+        self.coords = [0, 0] if coords is None else self.coords
```

Comment:
There is no need to have to line of `self.coords` definition, you could reduce the code: 
```suggestion
        self.coords = coords if coords else [0, 0]
```